mappings/net/minecraft/network/listener/ServerPlayPacketListener.mapping

```diff
@@ -88,3 +88,5 @@ CLASS net/minecraft/class_2792 net/minecraft/network/listener/ServerPlayPacketLi
 		ARG 1 packet
 	METHOD method_27273 onJigsawGenerating (Lnet/minecraft/class_5194;)V
 		ARG 1 packet
+	METHOD method_30303 onRecipeBookOption (Lnet/minecraft/class_5427;)V
```

Comment:
Should this maybe be like `onSetRecipeBookOptions`? (and the packet name too?)

Reply:
hmm, this packet sends the recipe book option for one recipe category (i.e. crafting, furnace, etc.)

imo the `set` part of the name you propose is extraneous.

Reply:
Okay right, I see. I think it might still make sense to have this be plural though, because there are still multiple options per category. In the case of the `set`, this is mainly motivated because `onRecipeBookOption` reads a bit weirdly with no verb

Reply:
So like `RecipeCategoryOption` then? it sends the option for like crafting, like two booleans of if the screen is open and if the filter craftable is on

Reply:
That’d be better, but I think it still would have the problems I mentioned before without the set and without the plural. Probably rather get a third opinion on this :)

Reply:
`onRecipeCategoryOption` seems good to me

Reply:
I still think the plural is better here. There are two options being set, gui open and filtering craftable. 2 options in 1 category is "category options".